### PR TITLE
added new NOTICE file Universal_Decoder_NOTICE 41545

### PR DESCRIPTION
**Pull request by roel4ez**

### CODE REVIEW

**Review by p-schuler — APPROVED** [review]

File: Samples/UniversalDecoder/NOTICE.txt [hunk 1]
```diff
@@ -2829,41 +2798,6 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 
 
----------------------------------------------------------
-
----------------------------------------------------------
-
-pino 7.5.1 - MIT
```
Comment (p-schuler) [inline Samples/UniversalDecoder/NOTICE.txt:2836]: weird - I see pino is bumped up from 7.6.2 to 7.6.3 in another PR: https://github.com/Azure/iotedge-lorawan-starterkit/pull/1232 - I would assume I should see the removal of pino 7.5.1. somewhere?

Comment (p-schuler) [reply Samples/UniversalDecoder/NOTICE.txt:2836]: @roel4ez - can you check that?

Comment (roel4ez) [reply Samples/UniversalDecoder/NOTICE.txt:2836]: yep looking into it

Comment (roel4ez) [reply Samples/UniversalDecoder/NOTICE.txt:2836]: 7.5.1 is removed here: https://github.com/Azure/iotedge-lorawan-starterkit/commit/02c722d99926ac33fb382963262324d7d2d3e4c3#diff-e73da8f41c646c40f89aa5581ad120ed0d90fc5c80ef05f1109e47bec4deeb44

However I expect to see a new 7.6.2 record in this notice. Still looking into it

Comment (roel4ez) [reply Samples/UniversalDecoder/NOTICE.txt:2836]: It is visible in the NOTICE when I check the CG, so I think it is a timing issue. I will run the pipeline again and I expect to see it show up

Comment (roel4ez) [reply Samples/UniversalDecoder/NOTICE.txt:2836]: #1257 fixes it

**Pull request closed without merging**